backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/storage/disk/image/RemoveImageCommand.java

```diff
@@ -295,6 +300,9 @@ private void removeImageMapping() {
     }
 
     private void performImageDbOperations() {
+        if (ACTIONS_NOT_REQUIRED_DB_OPERATION.contains(getParameters().getParentCommand())) {
```

Comment:
Have I?
If not contains -> performImageDbOperations() 
->
if contains -> do not performImageDbOperations()

Reply:
I did that because I moved performImageDbOperations() to both onSucceeded() and onFailed(), and thought that this condition better be inside the function to make things neater.
For now I have removed performImageDbOperations() from onFailed() until we figure out what to do here, but it will be used there, one way or the other.